templates/accounts/signup.html

```diff
@@ -22,12 +22,34 @@
         {% csrf_token %}
         <div class="neumorphic__card input-group input-group-lg mb-3">
           <span class="input-group-text"><i class="bi bi-person-fill"></i></span>
+          <input name="full_name" type="text" maxlength="50" class="form-control" id="id_full_name"
```

Comment:
As you are supposed to remove full_name and website fields in UserCreationForm, you should also remove these fields from template also.